src/main/java/teammates/common/datatransfer/InstructorAttributes.java

```diff
@@ -28,13 +29,8 @@
     public String role;
     public boolean isDisplayedToStudents;
     public String displayedName;
-    
-    /**
-     * The json representation of privileges, used for storing the 
-     * instructorPrivilege content in instructor entity
-     */
-    public String instructorPrivilegesAsText;
-    public transient InstructorPrivileges privileges;
+
+    public InstructorPrivileges privileges;
```

Comment:
Previously gson ignores `privileges` when converting `InstructorAttributes` into json string. Since we no longer store `InstructorPrivileges` as strings, the `transient` keyword can be removed. Making `privileges` not `transient` only affects how the `InstructorPrivileges` are stored in json files.